index.html

```diff
@@ -1002,6 +1006,7 @@
             <li><a href="https://github.com/w3c/pointerevents/pull/34">Clarify the button value for mouse drag</a></li>
             <li><a href="https://github.com/w3c/pointerevents/pull/24">Fix the touch-action processing model for zoom scenarios</a></li>
             <li><a href="https://github.com/w3c/pointerevents/pull/13">Add direction-specific touch-action values</a> (pan-left, pan-right, pan-up, pan-down) and clarified behavior of existing pan-x and pan-y values.</li>
+            <li><a href="https://github.com/w3c/pointerevents/pull/127">Add hasPointerCapture</a></li>
```

Comment:
the revision history is in reverse chronological order, so this should be at the very top of the list